src/general/OptionsMenu.cs

```diff
@@ -1142,12 +1169,21 @@ private void OnCustomUsernameTextChanged(string text)
         UpdateResetSaveButtonState();
     }
 
+    private void OnAudioOutputDeviceSettingSelected(int item)
+    {
+        Settings.Instance.SelectedAudioOutputDevice.Value = AudioOutputDevices[item];
+
+        Settings.Instance.ApplyAudioOutputDeviceSettings();
+        UpdateResetSaveButtonState();
+    }
+
     private void OnLanguageSettingSelected(int item)
     {
         Settings.Instance.SelectedLanguage.Value = Languages[item];
         resetLanguageButton.Visible = true;
 
         Settings.Instance.ApplyLanguageSettings();
+        UpdateDefaultAudioOutputDeviceText(audioOutputDeviceSelection);
```

Comment:
There's a bug with this, pressing the reset all settings doesn't apply the new language. Instead of having this call here and in OnResetLanguagePressed, this should be really in _Notification so that it reacts properly to any language change.